suppressible-error-prone/src/main/java/com/palantir/suppressibleerrorprone/SourceCodeUtils.java

```diff
@@ -63,5 +63,26 @@ private static int startPositionWithWhitespace(CharSequence sourceCode, int sour
         return pos + 1;
     }
 
+    static int removeUntil(CharSequence sourceCode, int endOfSuppression) {
+        int pos = endOfSuppression;
+
+        for (; pos < sourceCode.length(); pos++) {
+            char character = sourceCode.charAt(pos);
+            if (character == '\n' || !Character.isWhitespace(character)) {
+                break;
+            }
+        }
+
+        if (pos < sourceCode.length() && sourceCode.charAt(pos) == '\n') {
+            // If the next element is in a newline, remove the newline as well.
+            // Ideally, we also remove whitespace before the next element, but that would overlap with any done on the
+            // next element. We leave those spaces to the formatter.
```

Comment:
I'd leave this comment in the `LineRemovingReplacementFix`. It's more relevant there than this generic utility function.

Reply:
Yep!